contracts/locked/LockedSOV.sol

```diff
@@ -326,9 +345,8 @@ contract LockedSOV {
 	function _createVesting(address _tokenOwner) internal returns (address _vestingAddress) {
 		/// Here zero is given in place of amount, as amount is not really used in `vestingRegistry.createVesting()`.
 		vestingRegistry.createVesting(_tokenOwner, 0, cliff, duration);
-
 		_vestingAddress = _getVesting(_tokenOwner);
-		emit VestingCreated(_tokenOwner, _vestingAddress);
+		emit VestingCreated(msg.sender, _vestingAddress);
```

Comment:
Why is the address used here msg.sender and not tokenOwner?

Reply:
The event mentions the initiator, which could be another contract.

Maybe we can expand the event to have three parameters @jameshowlett977 as `_initiator, _userAddress, _vesting`

Reply:
i think an additional param would be good.

Reply:
done